### DISCUSSION TITLE
Caused by: java.sql.SQLException: IO Error: Extension "/root/.duckdb/extensions/v0.7.1/linux_amd64/httpfs.duckdb_extension" not found.

### DISCUSSION THREAD

**Discussion by deepuak** [duckdb/duckdb#7376] [General]
Hi Team,

I am using Java to load parquet files to duckDB and get the below error when trying to load parquet files from S3 bucket.

error : `Caused by: java.sql.SQLException: IO Error: Extension "/root/.duckdb/extensions/v0.7.1/linux_amd64/httpfs.duckdb_extension" not found.`
Can i run the below code in Java to install httpfs and resolve the error? **Btw, i am using duckDB-jdbc driver: 0.7.1**
Also please note that Java is run as a docker container service inside AWS ECS.
> Connection conn = DriverManager.getConnection("jdbc:duckdb:/tempdb1");
> 			Statement stmt = conn.createStatement();
> 			stmt.execute("INSTALL httpfs");
> 			stmt.execute("LOAD httpfs");

**Comment by Mytherin** [comment]
Installing and loading extensions should work in the JDBC client as well. Are you encountering an issue with installing and loading the HTTPFS extension?

**Reply by deepuak** [reply]
Hi @Mytherin 

I am now getting a different error and i have explained it here at https://github.com/duckdb/duckdb/discussions/7542

Can you please have a look and see what is going wrong?
